src/components/Proposals/ProposalDetails/ProposalDetails.js

```diff
@@ -79,15 +80,17 @@ function ProposalDetails({ vote }) {
               grid-gap: ${4 * GU}px;
             `}
           >
-            {youVoted && (
-              <div
-                css={`
-                  display: flex;
-                `}
-              >
+            <div
+              css={`
+                display: flex;
+                justify-content: space-between;
+              `}
+            >
+              <TargetAppBadge vote={vote} />
+              {youVoted && (
                 <Tag icon={<IconCheck size="small" />} label="Voted" />
-              </div>
-            )}
+              )}
```

Comment:
Just checking, are we keeping the "You voted" tag? I might be wrong but I thought this app badge might be replacing it.

Reply:
Hey, no. They both should be there. One is the target app and the other is the tag that appears when you already voted in the vote. :) like this voting app: 

![image](https://user-images.githubusercontent.com/10419340/93131133-f8a9b980-f6a9-11ea-90e4-f4baf8406906.png)